README.md

```diff
@@ -80,7 +80,8 @@ You can subscribe and join the mailing list on [Google Groups](https://groups.go
 
 ## IRC
 
-OCI discussion happens on #opencontainers on Freenode.
+OCI discussion happens on #opencontainers on Freenode
+([logs][irc-logs]).
```

Comment:
On Wed, Mar 30, 2016 at 02:19:03AM -0700, Qiang Huang wrote:
> > -OCI discussion happens on #opencontainers on Freenode.
> > +OCI discussion happens on #opencontainers on Freenode
> > +([logs][irc-logs]).
> 
> Why new line here?

Emacs paragraph-wrapping happens subconsciously ;).  Fixed in 52ffac6
→ ab61f5d.